package exceptions;

public class Finally2 {

	public static void main(String[] args) {
		
		Finally f =new Finally();
		f.test();

	}

}
